src/DataSet.cpp

```diff
@@ -245,7 +246,16 @@ int DataSet::readPsm(const std::string& line, const unsigned int lineNr,
     }
     proteins.swap(myPsm->proteinIds); // shrink to fit
   }
-  
+
+  if (label == -1) {
+    for(auto const& value: myPsm->proteinIds) { 
+    std::string token = value.substr(0, value.find("_") + 1);
+    
+    if (token != decoyPrefix) {
+      std::cerr << "Warning: Set decoy prefix don't match" << std::endl;
```

Comment:
Change the message to:
"Warning: The command line provided decoy prefix " << decoyPrefix << " don't match the decoy proteins prefix " << token << endl

Do so only if VERB > 1

Reply:
Fixed here: https://github.com/percolator/percolator/commit/b23a4d7f397e61647d38e1195823d331ac74a2de